revolv/templates/base/base.html

```diff
@@ -49,7 +49,7 @@
     {% endblock %}
 </head>
 
-<body {% block body_class %}{% endblock%}>
+<body onscroll="myfunction()" {% block body_class %}{% endblock%} id="{% if 'project' in request.get_full_path %}project{% else %}bye{% endif %}">
```

Comment:
Also, why is the `{% if 'project' in request.get_full_path %}project{% else %}bye{% endif %}` necessary? I'm certain we don't need the `bye`.